### PR TITLE
Minor change + some mess up

### PR DESCRIPTION
**Pull request by adehecq**
- fix a mess up due to conflicts with previous commits/branches. Some changes had been reverted, hence the many commits.
- fix an issue with a missing import

### CODE REVIEW

File: GeoUtils/proj_tools.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import rasterio as rio
 from rasterio.crs import CRS
 from shapely.geometry.polygon import Polygon
+from shapely.ops import transform
```
Comment (atedstone) [inline GeoUtils/proj_tools.py:7]: This is a confusing import given that `transform` elsewhere in the repo commonly refers to an Affine transform. Please consider referencing as shapely.ops.transform instead.

**Review by atedstone — CHANGES_REQUESTED** [review]
See line-specific comment.

File: GeoUtils/proj_tools.py (hunk 1 above)
Comment (adehecq) [reply GeoUtils/proj_tools.py]: Done!

**Comment by adehecq** [comment]
Also accidentally added to the PR a newer commit that adds a method get_bounds_projected to Raster.

**Review by atedstone — APPROVED** [review]

**Review by iamdonovan — APPROVED** [review]

**Pull request merged**